packages/react-mutation-mapper/src/component/oncokb/OncoKB.tsx

```diff
@@ -114,29 +122,80 @@ export default class OncoKB extends React.Component<IOncoKbProps, {}> {
                         />
                     </span>
                 );
-            } else {
-                oncoKbContent = (
+            }
+        }
+
+        return oncoKbContent;
+    }
+
+    private getAnnotationIcon(type: OncoKbCardDataType) {
+        if (
```

Comment:
I can update this, but I would suggest not wrapping the logic in the if statement, but rather directly return null when  !this.props.availableDataTypes?.has(OncoKbCardDataType[type])